readthedocs/projects/tasks.py

```diff
@@ -572,7 +569,7 @@ def update_imported_docs(version_pk):
 
 
 # Web tasks
-@task(queue='web')
+@shared_task(queue='web')
```

Comment:
I dont like using `shared_task` unless its a reusable app where you can not get the app. 
You can import the `app` from `worker` then use the decorator `task`
something like
```python
import readthedocs.worker import app

@app.task(queue='web')
```

Reply:
https://groups.google.com/forum/#!topic/celery-users/XiSDiNjBR6k

Reply:
Makes sense, I'll try the change.